### PR TITLE
Update OneBranch.Official.yml for Azure Pipelines

### PR DESCRIPTION
**Pull request by wenbingl**

### CODE REVIEW

**Comment by snnn** [comment]
Please replicate the change to the pull request pipeline yaml file: https://github.com/microsoft/onnxruntime-extensions/blob/main/.pipelines/OneBranch.PullRequest.yml , so that it can be run before this PR is merged.

**Comment by snnn** [comment]
Your pull request yaml file has grammar error.

File: .pipelines/OneBranch.Official.yml [hunk 1]
```diff
@@ -54,8 +54,7 @@ extends:
     - stage: linux_stage
       jobs:
       - job: linux_job
-        pool:
-          type: linux
+        pool: {vmImage: 'ubuntu-latest', name: 'Linux-CPU'}
```
Comment (snnn) [inline .pipelines/OneBranch.Official.yml:57]: No... you need to use their machine pool.

Comment (snnn) [reply .pipelines/OneBranch.Official.yml:57]: Ah... probably it is because the job itself is already in docker.

Comment (wenbingl) [reply .pipelines/OneBranch.Official.yml:57]: can we ask onebranch team for a solution, this is a very common problem.

Comment (snnn) [reply .pipelines/OneBranch.Official.yml:57]: I will give you their support mail list.

**Comment by wenbingl** [comment]
temporarily close this PR before we can find a solution for docker for Linux.

**Pull request closed without merging**